usagov/usagov-2021
# usagov-1294-content-manager-admin-permissions: new permissions

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1294

## Description

Benefit Finder Team Content needs permissions added in Drupal for work

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



### Requires New Config

### Requires New Content

### Validation Steps
- On http://localhost/admin/people/permissions check that Content Managers and Administrators have access to node benefit actions (i.e create, delete, edit, view etc) for benefit content types (i.e agency, benefit, criteria etc) 

## Security Review




## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
